sdk/monitor/opentelemetry-exporter-azuremonitor/opentelemetry/exporter/azuremonitor/_generated/_configuration.py

```diff
@@ -17,8 +17,8 @@
 
 VERSION = "unknown"
 
-class AzureMonitorExporterConfiguration(Configuration):
-    """Configuration for AzureMonitorExporter.
+class ApplicationInsightsClientConfiguration(Configuration):
```

Comment:
If these names are generated via the swagger shouldn't we be changing the names in the templates instead of changing it here? Wouldn't each language have to manually change the name if we do it this way?

Reply:
@lzchen This name is changed in the language README `swagger/README.md`